profile_library/paulmann licht/50396/model.json

```
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-11-15T11:59:34.923133",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "TS011F",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.15.4:docker"
+  },
+  "name": "LED Pear Smart Home Zigbee 3.0 LED Globe E27 806lm 7W Tunable White dimmable Opal",
+  "standby_power": 0
```

```suggestion
  "standby_power": 0.25
```

Standby power can never be 0. the light bulb will always consume a bit for zigbee radio even when off.
When you was unable to measure I suggest to set it to 0.25 which matches the standby consumption of some other paulmann lights and is a good estimation.